src/jit/morph.cpp

```diff
@@ -4967,7 +4967,7 @@ GenTreePtr Compiler::fgMorphMultiregStructArg(GenTreePtr arg, fgArgTabEntryPtr f
         if (lcl != nullptr)
         {
             // Its fields will need to accessed by address.
-            lvaSetVarDoNotEnregister(lcl->gtLclNum, DNER_IsStructArg);
+            lvaSetVarDoNotEnregister(lcl->gtLclNum DEBUG_ARG(DNER_IsStructArg));
```

Comment:
Should there be a comma?
i.e. `lvaSetVarDoNotEnregister(lcl->gtLclNum, DEBUG_ARG(DNER_IsStructArg));`

Reply:
No. Take a look at other `lvaSetVarDoNotEnregister` calls in this file.